var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -18,20 +18,21 @@ class Molgw(MakefilePackage):
     """
 
     homepage = "https://github.com/molgw/molgw"
-    url = "https://github.com/molgw/molgw/archive/v3.3.tar.gz"
+    url = "https://github.com/molgw/molgw/archive/v3.4.tar.gz"
```

Comment:
Spack is actually smart enough to perform URL inference based on the known version so this line doesn't need to be changed, but it is not a hard blocker to this PR.

Reply:
Thank you. I'll be aware of this for the next addition.